src/control/cmd/daos/filesystem.go

```diff
@@ -105,6 +107,10 @@ func (cmd *fsCopyCmd) Execute(_ []string) error {
 	cmd.Infof("    Files:       %d", ap.fs_copy_stats.num_files)
 	cmd.Infof("    Links:       %d", ap.fs_copy_stats.num_links)
 
+	if ap.fs_copy_stats.num_chmod_enotsup > 0 {
+		return errors.New(fmt.Sprintf("Copy completed successfully, but %d files had unsupported mode bits that could not be applied. Run with --ignore-unsupported to suppress this warning.", ap.fs_copy_stats.num_chmod_enotsup))
```

Comment:
`errors.Errorf("...", fargs)` should be used to create error from format string

Reply:
since this is a 2.4 backport, those change requests should be made on master first i guess

Reply:
+1, please :)

Reply:
ah, thanks for pointing that out, will do

Reply:
I pushed the changed here, and this PR for master: https://github.com/daos-stack/daos/pull/13729